x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/policy_template_form.tsx

```diff
@@ -138,19 +139,23 @@ const getGcpAccountTypeOptions = (isGcpOrgDisabled: boolean): CspRadioGroupProps
   },
 ];
 
-const getAzureAccountTypeOptions = (): CspRadioGroupProps['options'] => [
+const getAzureAccountTypeOptions = (
+  isAzureOrganizationDisabled: boolean
+): CspRadioGroupProps['options'] => [
   {
     id: AZURE_ORGANIZATION_ACCOUNT,
     label: i18n.translate('xpack.csp.fleetIntegration.azureAccountType.azureOrganizationLabel', {
       defaultMessage: 'Azure Organization',
     }),
-    disabled: true,
-    tooltip: i18n.translate(
-      'xpack.csp.fleetIntegration.azureAccountType.azureOrganizationDisabledTooltip',
-      {
-        defaultMessage: 'Coming Soon',
-      }
-    ),
+    disabled: isAzureOrganizationDisabled,
+    tooltip: isAzureOrganizationDisabled
+      ? i18n.translate(
+          'xpack.csp.fleetIntegration.azureAccountType.azureOrganizationDisabledTooltip',
+          {
+            defaultMessage: 'Coming Soon',
```

Comment:
nit: text should indicate it is available when you upgrade to the newer version

Reply:
sure ill push a change

Reply:
actually i suggest to get a text from product and push it in another pr in order to not hold back this merge